virttest/utils_test/qemu.py

```diff
@@ -142,20 +158,25 @@ def setup_win_driver_verifier(session, driver, vm, timeout=300):
     :param timeout: Timeout in seconds.
     :return: Session after reboot.
     """
-    verifier_setup_cmd = "verifier.exe /standard /driver %s.sys" % driver
-    if driver:
+    verifier_setup_cmd = "verifier /standard /driver %s.sys" % driver
```

Comment:
other parts look ok to me. @CongLi please help to review this path.

Thanks,
Xu